Packs/CVE_2025_31324/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2835 @@
+      description: Any results from the manual XQL?
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "104"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3725.5,
+          "y": 1504
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: Are there any SAP NetWeaver Visual Composer instances in your env?
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Please provide the agent IDs for those machines. If you have multiple endpoints, enter them as comma-separated values.
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: shortText
+        options: []
+        optionsarg:
+        - simple: "Yes"
+        - simple: "No"
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Have you find any instances of SAP NetWaver?
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "104":
+    id: "104"
+    taskid: 4493808b-53a4-42c3-868a-7e6abf7d6f83
+    type: regular
+    task:
+      id: 4493808b-53a4-42c3-868a-7e6abf7d6f83
+      version: -1
+      name: Set agent IDs to alert context
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setIncident
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "72"
+    scriptarguments:
+      agentsid:
+        complex:
+          root: Have you find any instances of SAP NetWaver?.Answers
+          accessor: "0"
+          transformers:
+          - operator: split
+            args:
+              delimiter:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3725.5,
+          "y": 1652
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "105":
+    id: "105"
+    taskid: e1abdc19-4408-4344-944d-124801e90411
+    type: regular
+    task:
+      id: e1abdc19-4408-4344-944d-124801e90411
+      version: -1
+      name: Get alert's extra data
+      description: Returns information about each alert ID.
+      script: '|||xdr-get-cloud-original-alerts'
```

Comment:
Can't remember, but is this the "get dynamic extra data" for Agent alerts ( and not cloud/analytics)?

Reply:
you needed the file_path and it 's provided in the response of get-cloud-original-alerts